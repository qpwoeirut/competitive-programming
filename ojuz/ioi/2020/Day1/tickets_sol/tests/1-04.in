df804a0e-e4b1-4b8d-a0f5-83401a72b5a3
80 1 1
593982665
321023513
482868026
426153594
598975331
279704644
504187707
997993130
835489161
5711838
383642542
896991209
133065342
425865263
149032351
462611275
259425073
900199227
258251923
888235498
305028903
521264131
27095365
42157235
730056566
798384208
321591658
153487565
641333230
116993600
475902639
223390883
415121490
817589073
216821648
830044100
338849570
915360403
293822505
49909311
822666699
418492556
747901727
162565692
746580954
708808918
454420599
534948842
259348018
846941575
86227918
379750164
847644684
86188350
353751473
380894979
670589254
600827918
14563849
329173072
543283760
885159951
32024634
491356250
229699202
565968376
349753865
501685195
350449657
134983152
300875009
408110051
958528075
823096872
838263670
336113
744706979
498677667
944970006
674346461
