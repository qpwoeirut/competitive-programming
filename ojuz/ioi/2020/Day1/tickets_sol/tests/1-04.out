131b432f-1f84-4f8f-9d55-41e868a6bc5d
OK
18919508441
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
